core/invoker/Dockerfile

```diff
@@ -11,6 +12,11 @@ rm -f docker-${DOCKER_VERSION}.tgz && \
 chmod +x /usr/bin/docker && \
 chmod +x /usr/bin/docker-runc
 
+# Install kubernetes client
+RUN wget --no-verbose https://storage.googleapis.com/kubernetes-release/release/v${KUBERNETES_VERSION}/bin/linux/amd64/kubectl && \
+chmod +x kubectl && \
+mv kubectl /usr/bin/kubectl
+
```

Comment:
I was expecting the client would talk directly to Kubernetes's API service using its REST API, not use kubectl.   I see that the KubernetesContainer has a mix of REST calls using the fabric.io client and kubectl commands.  Is due to ease of development, or is there a performance/functional reason to use kubectl to do the run command?

Reply:
This is just an artifact of copying the original Docker implementation that shelled out to `docker`. We had to replace usage of `kubectl` with the API for performance reasons in some places but it looks like we never got around to replacing it everywhere.

Reply:
makes sense; thanks.